fastlane/Fastfile

```diff
@@ -145,45 +131,19 @@ def build_code_current
   BUILD_CODE_FORMATTER.build_code(version: version)
 end
 
-# Returns the current internal build code of the app
-#
-def build_code_current_internal
-  # Read the current build code from the .xcconfig file and parse it into an AppVersion object
-  # The AppVersion is used because WP/JPiOS uses the four part (1.2.3.4) build code format, so the version
-  # calculator can be used to calculate the next four-part version
-  version = VERSION_FORMATTER.parse(INTERNAL_VERSION_FILE.read_build_code(attribute_name: 'VERSION_LONG'))
-  # Return the formatted build code
-  BUILD_CODE_FORMATTER.build_code(version: version)
-end
-
 # Returns the build code of the app for the code freeze. It is the release version name plus sets the build number to 0
 #
 def build_code_code_freeze
   # Read the current build code from the .xcconfig file and parse it into an AppVersion object
   # The AppVersion is used because WP/JPiOS uses the four part (1.2.3.4) build code format, so the version
   # calculator can be used to calculate the next four-part version
-  release_version_current = VERSION_FORMATTER.parse(INTERNAL_VERSION_FILE.read_release_version)
+  release_version_current = VERSION_FORMATTER.parse(PUBLIC_VERSION_FILE.read_release_version)
```

Comment:
I'm not sure why this was ever using the internal version file instead of the public one. So this change won't affect anything. The release version (e.g. 24.4) was the same between the public and internal files, anyway.